src/components/value/index.test.tsx

```diff
@@ -194,6 +235,34 @@ describe("<Value /> component functional testing", () => {
     }
   );
 
+  it("when dataType is datetime and datetime-local is unssuported, it calls the setDatetime with the correct value", () => {
+    jest.spyOn(helpers, "UseDatetimeBrowserSupport").mockReturnValue(false);
+    jest.spyOn(SolidFns, "getDatetime").mockImplementationOnce(() => null);
+
+    const mockSetter = jest
+      .spyOn(SolidFns, "setDatetime")
+      .mockImplementation(() => mockThing);
+
+    jest.spyOn(SolidFns, "saveSolidDatasetAt").mockResolvedValue(savedDataset);
+
+    const { getByTitle } = render(
+      <Value
+        dataType="datetime"
+        solidDataset={mockDatasetWithBdayResourceInfo}
+        thing={mockThing}
+        property={mockBdayPredicate}
+        edit
+        autosave
+        inputProps={{ title: "test title" }}
+      />
+    );
+    const input = getByTitle("test title");
+    input.focus();
+    fireEvent.change(input, { target: { value: "2020-12-30T12:30" } });
```

Comment:
This doesn't seem right - as I read this, this sets the value of the `date` input to a complete `datetime` string and ignores the `time` input? The test also doesn't validate that `mockSetter` actually was called with the correct value (which I'm assuming would indicate that the time part is being ignored?).